languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def whitelist_reader(path):
+    """Reads in whitelist file"""
+    with open(path, "r") as source:
+        for line in source:
+            yield line.rstrip().split()[0]
+
+
+def filter(tsv_path, phonemes, output_path):
+    """Creates tsv filtered by whitelist"""
+    with open(tsv_path, "r") as source:
+        with open(output_path, "w") as output:
+            for line in source:
+                pron = set(line.split("\t")[1].split())
+                if phonemes.issuperset(pron):
+                    print(line.rstrip(), file=output)
+                else:
+                    pass
+
+
+def main() -> None:
+    tsv_path = sys.argv[1]
+    iso639_code = tsv_path[tsv_path.rindex("/") + 1:tsv_path.index("_")]
```

Comment:
Can you add some comments about what is happening here? This is complex code.

I am also not sure we should have special handling for dialect; or rather, if there's a dialect specification, we should assume that the whitelist files are also dialect-specific. That should simplify the logic here quite a bit. And this seems to be hard-coded to use phonemic files, which is not intended.

Let me suggest a simpler setup: just have the user specify all three files as command-line arguments, at least for now. We can figure something else out when/if @lfashby incorporates this into `languages.json`.